Wave1_data/2.illumination_correction/scripts/0.perform_ic.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Run CellProfiler `illum.cppipe` (IC) pipeline
+#
+# In this notebook, we run the CellProfiler IC pipeline to calculate the illumination (illum) correction functions for all images per channel (5), apply the functions, and save images into a new directory.
```

Comment:
Just to confirm, 5 channels is correct? Recommend including what the channels are in this notebook or in a README.